src/main/scala/tech/cryptonomic/conseil/tezos/ApiOperations.scala

```diff
@@ -327,9 +331,7 @@ object ApiOperations extends DataOperations with MetadataOperations {
 
     dbHandle.run(fetchOperation).map{
       accounts =>
-        accounts.headOption.map { account =>
-          Map("account" -> account)
-        }
+        accounts.headOption.map(AccountResult(_))
```

Comment:
why not `accounts.headOption.map(AccountResult)`?

Reply:
Will check if the inference works, maybe I didn't even try this time (it failed so many times in my past experience 😢 )